src/services/transpile.ts

```diff
@@ -121,7 +122,7 @@ export function transpileModule(input: string, transpileOptions: TranspileOption
             languageVersion: getEmitScriptTarget(options),
             impliedNodeFormat: getImpliedNodeFormatForFile(toPath(inputFileName, "", compilerHost.getCanonicalFileName), /*packageJsonInfoCache*/ undefined, compilerHost, options),
             setExternalModuleIndicator: getSetExternalModuleIndicator(options),
-            jsDocParsingMode: JSDocParsingMode.ParseNone,
+            jsDocParsingMode: transpileOptions.jsDocParsingMode ?? JSDocParsingMode.ParseNone,

```

Comment:
I'm out and didn't get to fully review the final PR; I would consider that we make this `ParseAll` by default (rather, leave it undefined) as I had fully intended for there to be no breaks and for this to be entirely opt-in.